health_check/backends/base.py

```diff
@@ -29,15 +29,23 @@ class ServiceReturnedUnexpectedResult(HealthCheckException):
 
 class BaseHealthCheckBackend(object):
 
+
+    def description(self):
+        return None
```

Comment:
I'd propose to default to `self.__class__.__doc__` That way if you have a docstring on the HealthCheck class it will be used.